spring-5-security/src/main/java/com/baeldung/xss/XSSUtils.java

```diff
@@ -0,0 +1,21 @@
+package com.baeldung.xss;
+
+import org.jsoup.Jsoup;
+import org.jsoup.safety.Whitelist;
+import org.owasp.esapi.ESAPI;
+
+public class XSSUtils {
+    private XSSUtils() {
+    }
```

Comment:
Remove - not needed. I know some linters ask for this, but they're wrong!

Reply:
Done.